redis/3.2/README.md

```diff
@@ -1,7 +1,7 @@
 # REDIS 3.2
 
 ```Dockerfile
-FROM quay.io/continuouspipe/redis:3.2
+FROM quay.io/continuouspipe/redis:3.2_v1
```

Comment:
both the old one and the new one are wrong actually, its quay.io/continuouspipe/redis3:stable